Components/Decks/DraftDeckEditor.jsx

```diff
@@ -97,7 +97,8 @@ class DraftDeckEditor extends React.Component {
         if(!this.props.restrictedList && !this.props.currentRestrictedList) {
             deck.status = {};
         } else {
-            const selectedRestrictedList = restrictedList || this.props.currentRestrictedList;
+            const draftEventRestrictedList = this.props.restrictedList.filter(rl => rl._id === this.props.deck.eventId)[0];
```

Comment:
Thinking a bit, should we even be validating against a restricted list for draft decks? I think in the current admin for a draft event, there's no way to even select a RL. The only reason we'd need most of the code here is if we ever wanted to support a draft event that did have a RL.

Reply:
I agree that at the moment a restricted list would not be necessary. Using the code i committed (which looks for the RL of the event/draft event) leads to the same behaviour that you tested with locally. So i would say it wouldn´t hurt to use the event RL (which is always empty at the moment) so we have to change less stuff when we enable custom RL for draft events.
what do you think?